libs/compute_api/src/spec.rs

```diff
@@ -1,8 +1,8 @@
-//! `ComputeSpec` represents the contents of the spec.json file.
+//! `ComputeSpec` represents the `spec` key of the config.json file.
 //!
-//! The spec.json file is used to pass information to 'compute_ctl'. It contains
-//! all the information needed to start up the right version of PostgreSQL,
-//! and connect it to the storage nodes.
+//! The config.json file is used to pass information to 'compute_ctl'. It
+//! contains all the information needed to start up the right version of
+//! PostgreSQL, and connect it to the storage nodes.
```

Comment:
If I understand correctly, the `config.json` file might not include the `spec` portion, just the jwks keystore for authentication. In that case, it doesn't really contain enough information to connect to the storage nodes. That explanation is clearly still talking about just the `spec` part, not the `config.json` file as whole.

```suggestion
//! The ComputeSpec contains all the information needed to start up
//! the right version of PostgreSQL, and connect it to the storage nodes.
//! It can be passed as part of the `config.json`, or the control plane can
//! provide it by calling the compute_ctl's `/compute_ctl` endpoint, or
//! compute_ctl can fetch it by calling the control plane's API.
```